src/main/java/mate/academy/hibernate/relations/model/Movie.java

```diff
@@ -61,6 +79,7 @@ public String toString() {
         return "Movie{"
                 + "id=" + id
                 + ", title='" + title + '\''
+                + ", actors='" + actors + '\''
```

Comment:
Bad idea to keep all actor's `toString` in movie `toString`. Imagine you whould have bidirectional relation instead of uni - that means you whould have list of movies in actors. So, you'll have endless loop of toString actors in movies in actors in movies and so on. So, if you really need to keep list of your actors in `toString`, there should be list of their ids to avoid circuit. But it is never needed, so you can just remove actors from here.